### PR TITLE
Use static locations for Riot icons

### PR DESCRIPTION
**Pull request by turt2live**
See vector-im/riot-web#9009

~~**Requires riot.im changes - see timeline**~~ Deployed

### CODE REVIEW

File: changelog.d/4779.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+Swap out riot.im icons and logos for a more static resource.
```
Comment (richvdh) [inline changelog.d/4779.misc]: the icons and logos are staying the same - it's just the urls that are changing, no? Or am I misunderstanding?

Comment (turt2live) [reply changelog.d/4779.misc]: Some of them have changed to the new logo and colour scheme, but most are just moved.

**Comment by turt2live** [comment]
> I'd like to know why we aren't going to have the same problem again.

Up until this, riot-web and other random uploads in random directories on several domains were responsible for the various avatars. Instead of hoping that things don't change and aren't forgotten (one of the logos I replaced was a Vector logo), we now have the icons in a central place in version control in a dedicated directory. The risk at this point is forgetting to update the assets, but that's better than accidentally deleting something that another thing was using.

File: changelog.d/4779.misc (hunk 1 above)
Comment (richvdh) [reply changelog.d/4779.misc]: I'm also concerned that the mention of riot.im in our changelog will set alarm bells ringing, not without reason.

```suggestion
Update URLs for riot.im icons and logos in the default notification templates.
```

**Comment by richvdh** [comment]
> we now have the icons in a central place in version control in a dedicated directory.

ok, this certainly sounds like a sensible improvement. is there any scope for a README or something in that directory that reminds people that the URLs are used in notification emails, so to avoid breaking them if possible?

**Comment by turt2live** [comment]
Yup, already included :)

> The images contained here are used by email notifications and such - do not rename or delete them without replacing all their references in other projects (sydent, synapse, etc).

(see the riot.im PR referenced in the timeline earlier up)

**Pull request merged**